facet/README.md

```diff
@@ -0,0 +1,205 @@
+--- 
+layout: pattern
+title: Facet
+folder: facet
+permalink: /patterns/facet/
+categories:
+- structural
+language: en
+tags:
+- Decoupling
```

Comment:
I think the `Security` should also be added to this list, as mentioned lower in your readme, it is used for security purposes.

Reply:
The first security pattern!